Makefile.local

```diff
@@ -0,0 +1,17 @@
+# This is the local Makefile for the projectcalico/api repository, with targets
+# specific to github.com/projectcalico/api. This is opposed to Makefile, which
+# is mirrored from github.com/projectcalico/calico/api.
+
+# update pulls in the latest contents of this repository from the upstream
+# github.com/projectcalico/calico/api directory.
+CALICO_VERSION ?= $(shell git rev-parse --abbrev-ref HEAD)
+update:
+	rm -rf /tmp/calico-api-mirror
+	mkdir -p /tmp/calico-api-mirror
+	git clone --depth 1 git@github.com:projectcalico/calico.git -b $(CALICO_VERSION) /tmp/calico-api-mirror
+	cp -r /tmp/calico-api-mirror/api/. .
```

Comment:
Doesn't handle the case of a file being deleted, I think.

Reply:
I might have reached for `git read-tree` or something after pulling the monorepo as a remote.  But on reflection, why use an arcane incantation when `cp` will do :laughing: